src/System.Collections/src/System/Collections/Generic/QueueDebugView.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.Collections.Generic
+{
+    internal sealed class QueueDebugView<T>
```

Comment:
If the end result is the same, can we just use that then, rather than having a debug view specific to Queue?